manual_correction.py

```diff
@@ -458,27 +455,27 @@ def create_json(fname_nifti, name_rater, modified):
     if modified:
         metadata = {'Author': name_rater, 'Date': time.strftime('%Y-%m-%d %H:%M:%S')}
         fname_json = fname_nifti.rstrip('.nii').rstrip('.nii.gz') + '.json'
-        with open(fname_json, 'w') as outfile:
+        with open(fname_json, 'a') as outfile: # information regarding raters will be added to the file
```

Comment:
What is the difference between `w` and `a`?
Actually, ideally, we would like to append users to the existing JSON sidecar, see [here](https://github.com/spinalcordtoolbox/manual-correction/issues/34#issuecomment-1494663834).

Reply:
See https://github.com/spinalcordtoolbox/manual-correction/commit/db9af06cdcf8abffdd932ff4608d9aaa53adf627#r121437994

Regarding the formatting it's a bit different but I believe this is also clearer.

Reply:
When correcting the same label multiple times I am getting the following:

```
{
    "Author": "Jan Valosek",
    "Date": "2023-07-12 19:32:33"
}
{
    "Author": "Jan Valosek",
    "Date": "2023-07-12 19:34:00"
}
```

Is it valid JSON file?

Reply:
Seems not valid; valid JSON file should be:

```json
[
    {
        "Author": "Jan Valosek",
        "Date": "2023-07-12 19:32:33"
    },
    {
        "Author": "Jan Valosek",
        "Date": "2023-07-12 19:34:00"
    }
]
```

Reply:
This comment was addressed in https://github.com/spinalcordtoolbox/manual-correction/pull/43/commits/7f942bf3e25a7d57782c27453f60a4548334568b

Reply:
> A condition was also added to update our current json files see https://github.com/spinalcordtoolbox/manual-correction/pull/43/files#diff-5bb1b1bb5622bd39c9a9a13cd9e60d947bc3ba4d720998caccc37ed2b56a48a9R462-R464

This is cool!